### PR TITLE
No longer lowercases workspaces

### PR DESCRIPTION
**Pull request by patrickreck**
This solves #2

### CODE REVIEW

**Comment by dhuf** [comment]
+1 by reading and thanks for the pull request

File: Classes/Swisscom/Neos/CaseInsensitiveUrls/Aspect/LowercaseRoutePartAspect.php [hunk 1]
```diff
@@ -27,8 +27,14 @@ public function initializeObject() {
 	 * @return void
 	 */
 	public function convertRoutePartToLowerCase(\TYPO3\Flow\AOP\JoinPointInterface $joinPoint) {
-		$lowerCaseRequestPath = strtolower($joinPoint->getMethodArgument('requestPath'));
+		$requestPath = $joinPoint->getMethodArgument('requestPath');
 
+		if (strpos($requestPath, 'user')) {
```
Comment (jrenggli) [inline Classes/Swisscom/Neos/CaseInsensitiveUrls/Aspect/LowercaseRoutePartAspect.php:32]: Usage of `user` is fault-prone as this could break all URLs with the string `user` in uriSegments.
@patrickreck: Why did you delete the at sign? This would make it more distinctive.

**Comment by jrenggli** [comment]
Hi @patrickreck 

Thanks for your submission. It's nice to see, that someone else is using this package too and even contributes to it.

Please have a look at the comments in the source code.

Best,
Jonas

File: Classes/Swisscom/Neos/CaseInsensitiveUrls/Aspect/LowercaseRoutePartAspect.php (hunk 1 above)
Comment (patrickreck) [inline Classes/Swisscom/Neos/CaseInsensitiveUrls/Aspect/LowercaseRoutePartAspect.php:32]: When the NodeTree is built the API calls do not contain the at sign, thus not fixing the actual issue. 

Actual URLs with 'user' in them will be lowercase anyway after the migration's been run. The validation won't allow users to create new URLs containing uppercase letters.

**Comment by patrickreck** [comment]
Hey @jrenggli,

I've answered above :)
